data/borgmatic.d/config.yaml

```diff
@@ -1,34 +1,35 @@
-source_directories:
-    - /mnt/source
-repositories:
-    - path: /mnt/borg-repository
-one_file_system: true
-
-
-#   Passphrase is set in variable $BORG_PASSPHRASE
-#   encryption_passphrase: "DoNotMissToChangeYourPassphrase"
-compression: lz4
-archive_name_format: 'backup-{now}'
-
-
-keep_hourly: 2
-keep_daily: 7
-keep_weekly: 4
-keep_monthly: 12
-keep_yearly: 10
-
-
-
-checks:
-    - name: repository
-    - name: archives
-check_last: 3
-
-
-
-before_backup:
-    - echo "Starting a backup job."
-after_backup:
-    - echo "Backup created."
-on_error:
-    - echo "Error while creating a backup."
+location:
```

Comment:
FYI these section headers are deprecated in recent borgmatic versions.

Reply:
> FYI these section headers are deprecated in recent borgmatic versions.

This is what I get for not keeping up to date on the configs myself! Lemme check the relevant config and replace!

Reply:
I'll admit even I had systems with out-of-date borgmatic config files for an embarrassingly long time. :smile:

Reply:
@witten - This better?

https://github.com/borgmatic-collective/docker-borgmatic/pull/303/commits/d0953b44680848bc03383f499835d6c22d7bd6c7

Reply:
Yes, looks great! The only other thing you might get a deprecation warning on is the repositories list. If you change it to this, the warning will go away:

```yaml
repositories:
    - path: /mnt/borg-repository
```

I'd also recommend removing `prefix:` entirely. It's deprecated and `archive_name_format` supersedes it.

Reply:
Dun n dun! Shall let you review and close if good!

Reply:
LGTM!